js/classes/entities/midground/enemies/flying_snake.js

```diff
@@ -0,0 +1,199 @@
+/** 
+ * Flying snake !!
+ * (mostly just copy pasted from Snake)
+ * 
+ * @author Trae Claar
+ */
+class FlyingSnake {
+    /**
+     * Constructor for a Snake.
+     * 
+     * @param {Vector} pos the position at which the Snake should start
+     */
+    constructor(pos) {
+        this.base = new FlyingEnemyBase(
+            this, 
+            pos, 
+            FlyingSnake.SCALED_SIZE, 
+            FlyingSnake.SPEED, 
+            FlyingSnake.MAX_HEALTH, 
+            () => this.handleDeath(),
+            [Vector.add(pos, {x: 20, y: 0}), Vector.add(pos, {x: 500, y: 100}), Vector.add(pos, {x: 1000, y: -200})]
```

Comment:
What is up with these arbitrary vectors? Should we use some variables to make this more readable or does it make sense like this? Can you explain what is happening here?

Reply:
That is just an arbitrary path for testing out flying enemy base. The flying snake, it its current state, is not intended to be used in production. Probably should have left a note about that.